### PR TITLE
release: new version 2.1.1

### PR DESCRIPTION
**Pull request by membphis**
mainly about docs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,14 +19,35 @@
...
+- [2.1.1](#211)
 - [2.1.0](#210)
 - [2.0.0](#200)
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+
+# 2.1.1
+
+This release mainly adds test cases and bugfix for `manager api`.
```
Comment (membphis) [inline CHANGELOG.md:31]: `Manager API 2.1.1` should be used with `APISIX 2.1`. 

It is not recommended to use with other APISIX versions.

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,14 +19,35 @@
...
+
+### Test case
+
+Add e2e test cases for route, upstream, consumer, SSL, and plugins.
+
+## Bugfix
+ 
+- Fix: when created route by `admin api`, upstream can not be modified by dashboard #786
```
Comment (membphis) [inline CHANGELOG.md]: `admin api` to `Admin API`
`manager api` to `Manager API`

I think we should keep the same style.

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,14 +19,42 @@
...
+Add e2e test cases for route, upstream, consumer, SSL, and plugins.
+
+## Bugfix
+ 
+- Fix: when created route by `Admin API`, upstream can not be modified by dashboard. [#847](https://github.com/apache/apisix-dashboard/pull/847)
+- Fix: create route with jwt-auth will display an error. [#878](https://github.com/apache/apisix-dashboard/pull/878)
+- Fix: create route with error format remote_addrs `Manager API` will return 200, but should return 400. [#899](https://github.com/apache/apisix-dashboard/pull/899)
+- Fix: make field ID compatible with both string and int. [#902](https://github.com/apache/apisix-dashboard/pull/902)
+- Fix: can't run `Manager API` in intranet env. [#947](https://github.com/apache/apisix-dashboard/pull/947)
+- Fix: create a route through the dashboard, without filling in the client address, APISIX reports errors. [#948](https://github.com/apache/apisix-dashboard/pull/948)
```
Comment (juzhiyuan) [inline CHANGELOG.md:50]: The frontend's CHANGELOg have been added here.
cc @membphis

**Comment by liuxiran** [comment]
do we need to change the FE version in package.json? @juzhiyuan

**Comment by membphis** [comment]
> do we need to change the FE version in package.json? @juzhiyuan

and we need to update the version of backend @nic-chen

**Comment by membphis** [comment]
I think we need one tool to check all of the versions, here is the script of APISIX: 

https://github.com/apache/apisix/blob/master/utils/check-version.sh

**Comment by nic-chen** [comment]
> and we need to update the version of backend @nic-chen

version info for `Manager API` is not finished yet.

related issue: 
#853 

related pr: 
#900

**Comment by membphis** [comment]
> > and we need to update the version of backend @nic-chen
> 
> version info for `Manager API` is not finished yet.

ok, got it.

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -18,14 +18,41 @@
...
 - [2.0.0](#200)
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+
+# 2.1.1
+
+该版本主要增加了对 `Manager API` 和 APISIX 间的 E2E 测试， 以及 bug 修复。
+
+`Manager API` 2.1.1 应该与 APISIX 2.1 配合使用，不推荐与其他 APISIX 版本使用。
```
Comment (moonming) [inline CHANGELOG.zh-CN.md:32]: should also write in the README.md

Comment (juzhiyuan) [reply CHANGELOG.zh-CN.md:32]: image

1. extra space?
2. `它`?

Comment (nic-chen) [reply CHANGELOG.zh-CN.md]: updated.

File: docs/FAQ.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ Since the Dashboard caches the jsonschema data of the plugins in Apache APISIX,
 2. Execute the following commands.
 
 ```sh
-# `$version` is the version number of Apache APISIX, e.g. master or 2.0.
+# `$version` is the version number of Apache APISIX, e.g. master or 2.1.1.
```
Comment (juzhiyuan) [inline docs/FAQ.md:43]: Just to make sure, is this Apache APISIX's version or dashboard's branch?

cc @membphis @nic-chen

**Comment by membphis** [comment]
@nic-chen need you to process the conflict first, then we can merge this PR

**Review by juzhiyuan — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**